### PR TITLE
Stopped reloading the image when the request fails

### LINKED ISSUE #7: Don't reload the image if the request failed
Don't reload the image if the request failed

### PR DESCRIPTION
**Pull request by IonicaBizau**
Fixes #7.

- a small optimization
- refactoring
- fixed a typo
- coding style improvements

### CODE REVIEW

**Comment by IonicaBizau** [comment]
@silviubogan Can you please :fire: [this `try - catch`](https://github.com/jillix/jQuery-image-upload/blob/3e6c13db5bd59eae7a9b17b1ddbb34da6b6294a9/src/jQuery-image-upload.js#L244-L250)?

**Comment by silviubogan** [comment]
@IonicaBizau Done in this commit: bfe3a7a.

File: src/jQuery-image-upload.js [hunk 1]
```diff
@@ -344,17 +348,37 @@
...
+    /**
      * Trigger a callback when the selected images are loaded:
      * @param {String} selector
      * @param {Function} callback
      */
-    function imgLoad(selector, callback){
-        $(selector).each(function(){
+    function imgLoad (selector, callback) {
+        $(selector).each(function () {
             if (this.complete || /*for IE 10-*/ $(this).height() > 0) {
```
Comment (IonicaBizau) [inline src/jQuery-image-upload.js]: You could remove the second check. It was anyway a hack. I hope ~~IE~~ :hankey:  10 will be supported.

/cc @silviubogan

**Comment by IonicaBizau** [comment]
:ship: :fireworks: `1.1.0`

**Pull request merged**